tools/cldr-apps/src/main/java/org/unicode/cldr/web/WebContext.java

```diff
@@ -1334,6 +1334,22 @@
         return c;
     }
 
+    /**
+     * Clear a cookie
+     * @param request
+     * @param response
+     * @param c
+     */
+    public static void clearCookie(HttpServletRequest request, HttpServletResponse response, String c) {
+        Cookie c0 = WebContext.getCookie(request, c);
+        if (c0 != null) {
+            c0.setValue("");
+            c0.setPath("/");
+            // c0.setMaxAge(0);
+            response.addCookie(c0);
```

Comment:
Not sure how important this is. Also not sure if it would break if you try to use the server locally (not over https).  We don't have any sensitive information in cookies anymore with this change.